ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73.yaml

```diff
@@ -53,7 +53,7 @@ images:
   - name: ROX_PRODUCT_BRANDING
     value: RHACS_BRANDING
   dockerfile_path: Dockerfile.gen
-  from: ubi
+  from: ubi-minimal
```

Comment:
since 73's code freeze has already begun, I think we should wait until 74 for this

Reply:
This PR is mostly for 3.73 as we have CVS for that image and I'd like to fix them before the release